src/cuda/mod.rs

```diff
@@ -5,6 +5,7 @@
 
 use thiserror::Error;
 
+#[cfg(feature = "cuda")]
```

Comment:
Not here, the whole `cuda` module

Reply:
my bad... Also squashed the commits, ready now :)